File: Pug.Clients.Web/app/pug/pick-up-game.ts
import { Player } from "./player";
import { PugPhase } from "./phases/pug-phase";
import { Team } from "./team/team";

export interface PickUpGame {
    id: number;
    path: string[];
    currentPhase: PugPhase;
    teams: Team[];
    pickablePlayers: Player[];
}

File: Pug.Data.Models/Pugs/PickUpGame.cs
using MongoDB.Bson;
using Seaal.Data.MongoDB;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Pug.Data.Models.Pugs
{
    public class PickUpGame : IDocument<ObjectId>
    {
        public ObjectId Id { get; set; }
        public IEnumerable<string> Path { get; set; }
        public BasePugPhase CurrentPhase { get; set; }
        public IEnumerable<Team> Teams { get; set; }
    }
}
